### PR TITLE
[Doc-75] Docs for custom organization name when creating self-signed certificates

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /preview/yugabyte-platform/security/enable-encryption-in-transit/#create-custom-organization-name-in-self-signed-certificates

### CODE REVIEW

**Review by shristy — APPROVED** [review]
LGTM, thank you @aishwarya24

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 1]
```diff
@@ -38,6 +40,68 @@ YugabyteDB Anywhere can create self-signed certificates for each universe. These
 
 YugabyteDB Anywhere retains the root certificate and the root private key for all interactions with the cluster.
 
+### Create custom organization name in self-signed certificates
+
+YugabyteDB Anywhere creates self-signed certificates in some workflows such as create universe, where the organization name in all the certificates is set as `example.com` by default.
+
+If you are using YBA version 2.18.2 or later to manage universes with YugabyteDB version 2.18.2 or later, you can set a custom organization name using the global [runtime configuration](../../administer-yugabyte-platform/manage-runtime-config/) flag, `yb.tlsCertificate.organizationName`.
+
+Note that you need perform this customization before a self-signed certificate creation workflow such as Create universe for the change to take affect.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:49]:
```suggestion
Note that, for the change to take effect, you need to set the flag _before_ you run a workflow that generates a self-signed certificate.
```

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 2]
```diff
@@ -38,6 +40,68 @@ YugabyteDB Anywhere can create self-signed certificates for each universe. These
 
 YugabyteDB Anywhere retains the root certificate and the root private key for all interactions with the cluster.
 
+### Create custom organization name in self-signed certificates
+
+YugabyteDB Anywhere creates self-signed certificates in some workflows such as create universe, where the organization name in all the certificates is set as `example.com` by default.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:45]:
```suggestion
YugabyteDB Anywhere automatically creates self-signed certificates when you run some workflows, such as create universe. The organization name in certificates is set to `example.com` by default.
```

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 3]
```diff
@@ -38,6 +40,68 @@ YugabyteDB Anywhere can create self-signed certificates for each universe. These
...
+
+If you are using YBA version 2.18.2 or later to manage universes with YugabyteDB version 2.18.2 or later, you can set a custom organization name using the global [runtime configuration](../../administer-yugabyte-platform/manage-runtime-config/) flag, `yb.tlsCertificate.organizationName`.
+
+Note that you need perform this customization before a self-signed certificate creation workflow such as Create universe for the change to take affect.
+
+You can create a custom organisation name as follows:
+
+1. From your YugabyteDB Anywhere UI, navigate to **Admin** > **Advanced** and select the **Global Configuration** tab.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:53]:
```suggestion
1. In YugabyteDB Anywhere, navigate to **Admin** > **Advanced** and select the **Global Configuration** tab.
```

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 4]
```diff
@@ -38,6 +40,68 @@ YugabyteDB Anywhere can create self-signed certificates for each universe. These
 
 YugabyteDB Anywhere retains the root certificate and the root private key for all interactions with the cluster.
 
+### Create custom organization name in self-signed certificates
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:43]:
```suggestion
### Customize the organization name in self-signed certificates
```

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 5]
```diff
@@ -38,6 +40,68 @@ YugabyteDB Anywhere can create self-signed certificates for each universe. These
...
+
+YugabyteDB Anywhere creates self-signed certificates in some workflows such as create universe, where the organization name in all the certificates is set as `example.com` by default.
+
+If you are using YBA version 2.18.2 or later to manage universes with YugabyteDB version 2.18.2 or later, you can set a custom organization name using the global [runtime configuration](../../administer-yugabyte-platform/manage-runtime-config/) flag, `yb.tlsCertificate.organizationName`.
+
+Note that you need perform this customization before a self-signed certificate creation workflow such as Create universe for the change to take affect.
+
+You can create a custom organisation name as follows:
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:51]:
```suggestion
Customize the organization name as follows:
```

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 6]
```diff
@@ -38,6 +40,68 @@ YugabyteDB Anywhere can create self-signed certificates for each universe. These
...
+
+1. From your YugabyteDB Anywhere UI, navigate to **Admin** > **Advanced** and select the **Global Configuration** tab.
+1. In the **Search** bar, enter `yb.tlsCertificate.organizationName` to view the flag, as per the following illustration:
+
+    ![Custom Organization name](/images/yp/encryption-in-transit/custom-org-name.png)
+
+1. Click **Actions** > **Edit Configuration** to change the default Config Value (`example.com`), and click **Save**. Make sure to enter a string for the Config Value to have a custom organisation name.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:58]:
```suggestion
1. Click **Actions** > **Edit Configuration**, enter a new Config Value, and click **Save**.
```

**Review by ddhodge — APPROVED** [review]
Misc edits. Port to stable.

**Pull request merged**